internal/venice-common/src/main/java/com/linkedin/venice/system/store/MetaStoreWriter.java

```diff
@@ -360,6 +365,10 @@ private void update(
     });
   }
 
+  HelixReadOnlySchemaRepository getSchemaRepository() {
```

Comment:
So we don't clean up the unused schemas from meta store?

Reply:
I dont recall why I skipped system-stores, but given system schema are rarely updated, it might be ok to skip them ?

Reply:
Actually, I was referring to the following question:
When we cleanup unused schema from backend/ZK, do we clean these schema from meta store for the corresponding Venice store as well?

Reply:
It stores schema per store version, so when its deleted from zk, it wont be calling the delete method again.

Reply:
I understood about the used schema tracking in meta store.
In meta store, we also store all the schemas and if we decide to delete unused schemas from ZK, we should delete these schemas from meta store as well, right?

Reply:
Good catch. updated to delete meta store schemas